src/assets/styles/footer.scss

```diff
@@ -0,0 +1,89 @@
+.footer {
+  display: inline-block;
+  background: $light-gray;
+  height: 10em;
+  width: 100%;
+
+  .footer-buttons {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  a.btn {
+    margin: 1.5em 0.6em 0.3em 0.6em;
+    color: white;
+    text-decoration: none;
+    flex-grow: 1;
+    font-size: 1em;
+    text-align: center;
+    position: relative;
+    min-width: calc(50% - 2.5em);
+  }
+
+  .privacy-policy {
+    margin: 0 1em;
+    &:link, &:visited, &:hover {
+      color: $dark;
+    }
+  }
+
+  .host-btn {
+    background: $aclu-blue;
+  }
+
+  .share-btn {
+    background: $facebook-blue;
+
+    &:hover {
+      background: lighten($facebook-blue, 8%);
+    }
+    span {
+      margin-right: .2em;
+    }
+    img {
+      height: 1.1em;
+      width: 1.1em;
+      vertical-align: sub;
+    }
+  }
+
+  .privacy-policy {
+    margin-top: 0.2em;
+    font-size: 0.7em;
+    display: block;
+  }
+}
+
+.event-map.-active ~ .footer {
+  position: fixed;
+  background: transparent;
+  bottom: 1em;
+  height: auto;
+}
+
+@media (min-width: $breakpoint-large) {
+  .event-map.-active ~ .footer,
+  .event-map ~ .footer {
+    left: 0;
+    position: fixed;
+    background: transparent;
+    bottom: 1em;
+    height: auto;
```

Comment:
I noticed a bug here: because the footer still has 100% width, it extends to the right across the map and event list, and you then can't interact with anything it's covering, for example an RSVP button in the event list. `width: auto` should do the trick I think.